format/Message.fbs

```diff
@@ -68,6 +68,11 @@ table Utf8 {
 table Binary {
 }
 
+table FixedWidthBinary {
+  /// Number of bytes per value
+  byteWidth: int;
+}
```

Comment:
Utf8 code points use a variable number of bytes (1 to 4), so to fully account for fixed width unicode, you would have to store (N code points) * 4 bytes. This is likely to be wasteful compared with the current list-like / variable-length UTF8 representation